lib/cyclone-server/src/tower.rs

```diff
@@ -1,5 +1,3 @@
-pub use limit_requests::{LimitRequest, LimitRequestLayer};
```

Comment:
Why are these type re-exports removed?

Reply:
Clippy seems to believe the import is unused. Does not mean that we don't want to keep though, but I see I've seen it on `main` for a couple weeks now.
```
warning: unused import: limit_requests::{LimitRequest, LimitRequestLayer}
```